### PR TITLE
DS-659 Allow elements in RadioOption helpText

### PR DESCRIPTION
**Pull request by david-cho-lerat-sonarsource**

### CODE REVIEW

File: stories/RadioButtonGroup-stories.tsx
Comment (gregaubert) [inline stories/RadioButtonGroup-stories.tsx:1]: We shouldn't have to replace the stories, a string is a valid ReactNode

**Review by gregaubert — APPROVED** [review]

**Review by jeremy-davis-sonarsource — APPROVED** [review]
Nice!

File: stories/RadioButtonGroup-stories.tsx
Comment (jeremy-davis-sonarsource) [reply stories/RadioButtonGroup-stories.tsx:1]: They're just a couple of examples

Comment (david-cho-lerat-sonarsource) [reply stories/RadioButtonGroup-stories.tsx:1]: Yes, some `helpText` are left as `string`, and some use a `ReactNode`, just to reflect that we can.

**Pull request merged**